### PR TITLE
LPS-177991 Added button to refresh loading page and the function to refresh loading page.'

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowOutlet.tsx [hunk 1]
```diff
@@ -127,6 +128,15 @@ const TestflowOutlet = () => {
 		return null;
 	}
 
+	if (testrayTask.dueStatus.key === TaskStatuses.PROCESSING) {
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowOutlet.tsx:131]: This should be PROCESSING or OPEN

I can try to forward this and then you send a follow up, if CI fail, you ammend this change

Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowOutlet.tsx:131]:
```suggestion
	if ([TaskStatuses.PROCESSING, TaskStatuses.OPEN].includes(testrayTask.dueStatus.key)) {
```

**Review by kevenleone — APPROVED** [review]
LGTM 🔥 

A OPEN status is missing in component check

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**